src/function.ts

```diff
@@ -140,6 +169,19 @@ abstract class FunctionBase<P, O>
       ResultSelector: "$.Payload",
     };
   }
+
+  public readonly eventBus = makeEventBusIntegration<
+    P,
+    FunctionEventBusTargetProps | undefined
+  >({
+    target: (props, targetInput) =>
+      new aws_events_targets.LambdaFunction(this.resource, {
+        deadLetterQueue: props?.deadLetterQueue,
+        maxEventAge: props?.maxEventAge,
+        retryAttempts: props?.retryAttempts,
```

Comment:
Nit: 
```ts
...props
```

More concise and automatically accounts for the case when new fields are added.